src/helpers/accessiblity.ts

```diff
@@ -160,3 +164,33 @@ export function getAccessibilityState(element: ReactTestInstance) {
     selected: ariaSelected ?? accessibilityState?.selected,
   };
 }
+
+export function getAccessibilityValue(
+  element: ReactTestInstance
+): AccessibilityValue | undefined {
+  const {
+    accessibilityValue,
+    'aria-valuemax': ariaValueMax,
+    'aria-valuemin': ariaValueMin,
+    'aria-valuenow': ariaValueNow,
+    'aria-valuetext': ariaValueText,
+  } = element.props;
+
+  const hasAnyAccessibilityValueProps =
```

Comment:
nit: here I think that a condition hasNoAccessibilityValueProps would be a bit clearer because it would remove the negation in the next if